src/Database/LSMTree/Internal/Serialise/RawBytes.hs

```diff
@@ -146,6 +148,12 @@ sizeofRawBytes (RawBytes pvec) = P.length pvec
 concat :: [RawBytes] -> RawBytes
 concat rbs = RawBytes (P.concat (fmap toVector rbs))
 
+fromList :: [Word8] -> RawBytes
+fromList = coerce P.fromList
+
+toList :: RawBytes -> [Word8]
```

Comment:
I'd name these `pack` and `unpack` (as in `Text` and `ByteString`). `toList` clashes with `Foldable` method, it's a bad name when names are unqualified.

(Alternatively or additionally, `RawBytes` could have instance of `GHC.Exts.IsList`)